tools/install.ps1

```diff
@@ -0,0 +1,45 @@
+[CmdletBinding()]
+param (
+)
+​
+Set-StrictMode -Version 3.0
+$ErrorActionPreference = "Stop"
+​
+$org = 'puppetlabs'
+$repo = 'pdkgo'
+​
+$app = 'pct'
+​
+$arch = "x86_64"
+$os = 'windows'
+$ext = '.zip'
+​
+$ver = (Invoke-RestMethod "https://api.github.com/repos/${org}/${repo}/releases")[0].tag_name
```

Comment:
```suggestion
$ver = (Invoke-RestMethod "https://api.github.com/repos/${org}/${repo}/releases")[0].tag_name
$ver = $ver -replace '-pre',''
```

Reply:
Do we want to remove `-pre`? The download URL still contains it: https://github.com/puppetlabs/pdkgo/releases/download/0.1.0-pre/pct_0.1.0-pre_windows_x86_64.zip

Reply:
Good point!